### DISCUSSION TITLE
contact/circle

### DISCUSSION THREAD

**Discussion by giscus** [SMGoro/issues-json-generator#7] [General]
# contact/circle

http://localhost:4000/contact/circle

**Comment by SMGoro** [comment]
a

**Comment by 2013-GH** [comment]
好久不见，SummerGoro

**Comment by 2013-GH** [comment]
哎呦免费域名不行，吓哭了

**Comment by 2013-GH** [comment]
谢谢你哈嘿嘿，我还是想学怎么弄上一个Discord

**Comment by 2013-GH** [comment]
呃啊！SMGoro！！！你Discord服务器的网页版怎么拦我啊！！！还有，只有无痕看到的东西比较全
